src/main/resources/headers/okapi-token.yaml

```diff
@@ -0,0 +1,7 @@
+in: header
+name: X-Okapi-Token
+description: Okapi Token
+# The /_/tenant route will not have this header.
```

Comment:
Which .yaml file contains the `/_/tenant` route?

Reply:
Also good catch. No file does. I need to add it.

Reply:
I think we may not want to replicate the yaml, but instead perhaps just note where to find the yaml for the /_/tenant route. It's here: https://github.com/folio-org/folio-vertx-lib/blob/master/core/src/main/resources/openapi/tenant-2.0.yaml. Duplicating may lead to bugs (not even sure if it would work). If we can make it clear in the comments what is going on (it's passing through the request to a handler in a dependent library folio-vertx-lib).